migrations/20240221100230_add_issued_at_to_flow_state.up.sql

```diff
@@ -0,0 +1 @@
+alter table {{ index .Options "Namespace" }}.flow_state add column if not exists issued_at timestamptz null;
```

Comment:
i think this needs to be wrapped in a `do $$ begin ... end $$` block

Reply:
Good catch, thanks! Let me update the timestamp as well